lib/main/atom/autoCompleteProvider.ts

```diff
@@ -109,6 +111,7 @@ loadSnippets();
 
 export var provider: autocompleteplus.Provider = {
     selector: '.source.ts',
+    inclusionPriority: 1,
```

Comment:
Ahh, I remembered that the fuzzy suggestions and accurate suggestions are mixed together (there used to be an issue about it? but I can't find it last night), and thought adding this fixed the order. But, it doesn't seem to be necessary.